addons/medical/CfgVehicles.hpp

```diff
@@ -328,22 +328,23 @@ class CfgVehicles {
         author = "$STR_ACE_Common_ACETeam";
         class Arguments {
             class EnableList {
-                displayName = "List";
-                description = "List of vehicles that will be classified as medical vehicle, separated by commas.";
+                displayName = "$STR_ACE_AssignMedicVehicle_EnableList_DisplayName";
+                description = "$STR_ACE_AssignMedicVehicle_EnableList_Description";
                 defaultValue = "";
                 typeName = "STRING";
             };
             class enabled {
-                displayName = "Is Medical Vehicle";
-                description = "Whatever or not the objects in the list will be a medical vehicle.";
+                displayName = "$STR_ACE_AssignMedicVehicle_enabled_DisplayName";
+                description = "$STR_ACE_AssignMedicVehicle_enabled_Description";
+
                 typeName = "NUMBER";
                 class values {
                     class none {
-                        name = "No";
+                        name = "$STR_ACE_Medical_No";
                         value = 0;
                     };
                     class medic {
-                        name = "Yes";
+                        name = "$STR_ACE_Medical_Yes";
                         value = 1;
```

Comment:
As @Glowbal requested these specific options use typeName = "NUMBER" so I had to stringtable them.